tests/interface/test_panes.py

```diff
@@ -1707,7 +1707,7 @@ def test_PythonProcessPane_start_process_user_environment_variables():
     ):
         mock_sys.platform = "darwin"
         ppp = mu.interface.panes.PythonProcessPane()
-        envars = [["name", "value"]]
+        envars = {"name": "value"}
```

Comment:
Could we also add a test like this were we load a list with repeated values and assert that we still get a single entry?
I've run a manual tests locally by editing the `session.json` file, but it'd be good to have it in the CI as well.

Reply:
Thanks, done!